ogc/src/main/scala/geotrellis/server/ogc/wcs/WcsParams.scala

```diff
@@ -52,19 +52,20 @@ case class DescribeCoverageWcsParams(version: String, identifiers: Seq[String])
  * Parameters descriptions can be also found here: https://mapserver.org/ogc/wcs_server.html
  */
 case class GetCoverageWcsParams(
-                                 version: String,
-                                 identifier: String,
-                                 boundingBox: Extent,
-                                 temporalSequence: List[OgcTimeInterval],
-                                 format: OutputFormat,
-                                 gridBaseCRS: CRS,
-                                 gridCS: URI,
-                                 gridType: URI,
-                                 // GridOrigin is BBOX minx, maxy // swapped in case of a geographic projection
-                                 gridOrigin: (Double, Double),
-                                 // GridOffsets is xres, yres // swapped in case of a geographic projection
-                                 gridOffsets: (Double, Double),
-                                 crs: CRS
+  version: String,
+  identifier: String,
+  boundingBox: Extent,
+  temporalSequence: List[OgcTimeInterval],
+  format: OutputFormat,
+  gridBaseCRS: CRS,
+  gridCS: URI,
+  gridType: URI,
+  // GridOrigin is BBOX minx, maxy // swapped in case of a geographic projection
+  gridOrigin: (Double, Double),
+  // GridOffsets is xres, yres // swapped in case of a geographic projection
+  gridOffsets: (Double, Double),
+  crs: CRS,
+  params: ParamMap
```

Comment:
The cost was to expose `ParamMap` into the user API so users would have access to all the input params.